src/main/java/org/simulator/io/CSVImporter.java

```diff
@@ -54,19 +48,63 @@
   private static final transient Logger logger = Logger.getLogger(CSVImporter.class.getName());
 
   /**
-   * The header of the file
+   * This method reads the data from the CSV file and converts it
+   * into the map with key as the column name and value as the array of amounts
+   * for the particular column.
+   * 
+   * @param pathname The path of the CSV file
+   * @return columnsMap -> LinkedListHashMap
    */
-  private String[] header;
+  private Map<String, double[]> readDataFromCSV(String pathname) throws IOException {
+    Map<String, double[]> columnsMap = new LinkedHashMap<>();
+    BufferedReader reader = new BufferedReader(new FileReader(pathname));
+    String line = reader.readLine();
+    List<String> lines = new LinkedList<String>();
+    String[] identifiers = line.split(",");
+    if (line != null) {
+      line = reader.readLine();
+      while ((line != null) && !line.isEmpty()) {
+        lines.add(line);
+        line = reader.readLine();
+      }
+
+      if (identifiers[0].equalsIgnoreCase("time")){
```

Comment:
This `String` literal `"time"` should be defined somewhere as a `public static final String TIME = "time"`.